src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -147,8 +147,8 @@ const ReleaseContent = () => {
               <Details summary="Download associated files">
                 <ul className="govuk-list govuk-list--bullet">
                   {release.downloadFiles.map(
-                    ({ id, extension, name, fileName, path, size }) => (
-                      <li key={path}>
+                    ({ id, extension, name, fileName, size }) => (
+                      <li key={fileName}>
```

Comment:
Wouldn't `id` be better here?

Reply:
Ah yes, not sure why I didn't use id to begin with here. I might have been thinking of the 'All files' zip problem at the time, but that file doesn't appear in the Admin. Thanks.